pages/byoc/installation.mdx

```diff
@@ -0,0 +1,109 @@
+# BYOC Installation
+
+We offer two installation methods, depending on whether you are using kubernetes or vm. Both are very simple and you can deploy in less than 1 minute! they all need agent token, you can get it from ModelZ dashboard.
+![agent_token](./images/agent_token.jpg)
+## VM
```

Comment:
Is this method also suitable for a physical machine? If so, could we explain it could be a cloud vm or your home lab, or a single machine like in `architecture`.

Reply:
Yep, the physical machine is suitable, change VM to be a more common case LGTM.